Add more considerations/ steps to Software upgrade guide (#1137)
astronomer/docs: software/install-cli.md

<pre><code parentName="pre">{`% astro version
Astro CLI Version: ${siteVariables.cliVersion}`}</code></pre>

> 

Author: ```suggestion
```

## Upgrade the CLI

<Tabs
    defaultValue="mac"
    groupId= "upgrade-the-cli"
    values={[
        {label: 'Mac', value: 'mac'},
        {label: 'Windows', value: 'windows'},
        {label: 'Linux', value: 'linux'},
    ]}>
<TabItem value="mac">

> To upgrade the Astro CLI to the latest version, you can run the same command as you did to install the CLI for the first time:

Reviewer: ```suggestion
To upgrade the Astro CLI to the latest version, run the following command:
```

```sh
brew install astro
```

</TabItem>

<TabItem value="windows">

To upgrade the Astro CLI on Windows:

1. Delete the existing `astro.exe` file on your machine.

> 2. Go to the [**Releases** page of the Astro CLI GitHub repository](https://github.com/astronomer/astro-cli/releases). Based on the version of the CLI you want and your CPU architecture, download one of the `.zip` files available on this page.

Reviewer: ```suggestion
2. Go to the [Astro CLI Releases page](https://github.com/astronomer/astro-cli/releases) and then download an Astro CLI `.zip` file. Astronomer recommends downloading the latest version to take advantage of the latest features and functionality.
```
I don't think _CPU architecture_ is necessary or relevant here.

Author: That's not quite right, you need to choose a download with either `amd` or `arm` which is based on your CPU architecutre
![Screen Shot 2022-10-28 at 12 02 45 PM](https://user-images.githubusercontent.com/74574233/198572395-9fd63554-d294-4b92-b3ab-15240f12ae1d.png)

> 2. Go to the [**Releases** page of the Astro CLI GitHub repository](https://github.com/astronomer/astro-cli/releases). Based on the version of the CLI you want and your CPU architecture, download one of the `.zip` files available on this page.

Author: ```suggestion
2. Go to the [**Releases** page of the Astro CLI GitHub repository](https://github.com/astronomer/astro-cli/releases). Based on the version of the CLI you want and your CPU architecture, download one of the `.zip` files available in the **Assets** menu.
```

>      For example, to upgrade to v1.0.0 of the Astro CLI on a Windows machine with an AMD 64 architecture, you download `astro_1.0.0-converged_windows_amd64.zip`.

Reviewer: ```suggestion
     For example, to upgrade to v1.0.0 of the Astro CLI on a Windows machine with an AMD 64 architecture, you download `astro_1.0.0-converged_windows_amd64.zip`.
```
Again, this doesn't seem relevant. From what I can tell, the zip files are organized by version only.

> 3. If the `.zip` file does not automatically unzip, run the following command to unzip the executable:

Reviewer: ```suggestion
3. If the `.zip` file isn't automatically extracted, run the following command to extract the executable:
```
![image](https://user-images.githubusercontent.com/104205257/188635446-dfe56277-17d8-4913-affd-89914465dea2.png)
